Purchases/Attribution/RCAttributionFetcher.m

```diff
@@ -13,23 +13,15 @@
 #import "RCIdentityManager.h"
 #import "RCBackend.h"
 #import "RCAttributionData.h"
+#import "RCSystemInfo.h"
 @import PurchasesCoreSwift;
 
-
-@protocol FakeAdClient <NSObject>
-
-+ (instancetype)sharedClient;
-- (void)requestAttributionDetailsWithBlock:(RCAttributionDetailsBlock)completionHandler;
-
-@end
-
-
-@protocol FakeASIdentifierManager <NSObject>
-
-+ (instancetype)sharedManager;
-
-@end
-
+typedef NS_ENUM(NSUInteger, FakeATTrackingManagerAuthorizationStatus) {
+    FakeATTrackingManagerAuthorizationStatusNotDetermined = 0,
+    FakeATTrackingManagerAuthorizationStatusRestricted,
+    FakeATTrackingManagerAuthorizationStatusDenied,
+    FakeATTrackingManagerAuthorizationStatusAuthorized
+};
```

Comment:
Do you think this enum should live here or in the RCAttributionTypeFactory like the other "fake" classes

Reply:
good catch, forgot to move that one over